pkgs/games/starruler2/engine.nix

```diff
@@ -0,0 +1,81 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bzip2,
+  cmake,
+  curl,
+  freetype,
+  glew,
+  libGLU,
+  libogg,
+  libpng,
+  libvorbis,
+  libXi,
+  libXrandr,
+  libXxf86vm,
+  openal,
+  which,
+  zlib,
+}:
+stdenv.mkDerivation {
+  pname = "starruler2-engine";
+  version = "2022-05-27";
```

Comment:
why not space between `version` and `src` ? There's one in `data.nix`. I personally prefer with space, it makes it more readable to me

Reply:
Done.